### ISSUE TITLE
Preview is stuck on Booting webcontainer for github connected typescript blank project

### ISSUE THREAD

**Issue by akoni50** [stackblitz/core#2847]
### Describe the bug

Created a new typescript blank project. Default editor is launched, at this point preview is perfectly working. Then proceed to create repository from stackblitz with default options. Repository is created and stackblitz reloads after which preview says stuck on Booting WebContainer. 

### Link to the blitz that caused the error

https://stackblitz.com/~/github.com/akoni50/typescript-aeiake

### Steps to reproduce

Created a new typescript blank project. Default editor is launched, at this point preview is perfectly working. Then proceed to create repository from stackblitz with default options. Repository is created and stackblitz reloads after which preview says stuck on Booting WebContainer. 

### Expected behavior

Preview should work after creating and connecting to a github repository

### Parity with Local

- [X] I have [run the project in my local machine](https://github.com/stackblitz/webcontainer-core/blob/main/repro.md#try-locally) and I could not reproduce the issue.

### Platform

```
Version = 1.84.0
Hash = 4f3918d67b0d0d6d912f5928adca1fb73bf3c67b
WebContainer = 6a615fc0

Browser name  = Chrome
Full version  = 119.0.0.0
Major version = 119
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 108577503,
  "usedJSHeapSize": 94792003,
  "jsHeapSizeLimit": 4294705152
}
```

### Additional context

Note this doesn't happen with other starter projects like React. Only noticed with typescript blank project

**Comment by mike-source** [comment]
I have the same issue... I'm not sure if this is a bug or if the documentation is just hard to follow.

This was my first time using stackblitz, someone sent me a url, everything worked and the app ran in the RHS with no config. 

Created a repo on my github using the stackblitz gui, opened it back up using the `https://stackblitz.com/~/github.com/user/repo` url, it now stops on "booting webcontainer". Is not clear from docs how the envrionment was compiling the TS or running the app before. No clues in the package.json or tsconfig as to how this all works.

**Comment by p1errot** [comment]
Same issue here, my repo is `https://stackblitz.com/~/github.com/p1errot/css-layers`

**Comment by PierreENOlivier** [comment]
Same issue here. The preview fails after creating the repo.
https://stackblitz.com/~/github.com/PierreENOlivier/boolean-coding-week

**Comment by HeyGarrison** [comment]
@daaanny90 do you happen to have a link to the project that was causing you issues?

**Comment by daaanny90** [comment]
@HeyGarrison https://stackblitz.com/~/github.com/daaanny90/POC-overlay-logic

**Comment by HeyGarrison** [comment]
@daaanny90 are you still able to consistently reproduce this?

I'm using and Arc and can't reproduce it for the life of me.

**Comment by binerdy** [comment]
In my case it was a browser setting that prevented loading execution of the web assembly.
https://developer.stackblitz.com/platform/webcontainers/browser-config

This fixed the problem for me.

**Comment by angelalaw01887** [comment]
having the same problem first time using stackblitz and I am getting Booting WebContainer not able to get past this...
If I create the project for the first time I get the preview area no problem.  But if I log out and come back into the project I get that Booting WebContainer message.  This is a static project with html/js/css.  

How do I get past this?

**Comment by Bharani77** [comment]
![image](https://github.com/user-attachments/assets/2b7d6e23-9978-4ff3-aad5-7d76deff2beb)
Please check and fix the Web container boot issue.. It is working in firefox (beta ) but not in chrome. have tried all workaround step none gets fixed.
